components/api/src/main/java/com/hotels/styx/api/Clocks.java

```diff
@@ -26,14 +26,15 @@ private Clocks() {
     }
```

Comment:
I don't think we should simply remove the comment text on these methods to the "@return" part. They should have different roles. The main comment text should help make clear when someone would use the method, the "@return" should be a short description of the return value.